File: Javascript/Mandelbrot.js
            c=(R,I)=>{return {r:R,i:I}}
       p=(t,z)=>c(t.r+z.r,t.i+z.i);m=(t,z)=>
      c(t.r*z.r-t.i*z.i,t.r*z.i+t.i*z.r);s=t=>
    Math.sqrt(t.r*t.r+t.i*t.i);function M(C) {
  z=[c(0,0)];for(var i in Array(200).fill()){var q=z[i];
    v=p(m(q,q),C);if(s(v)>2)break;else z.push(v)}
      return z.length!=201}for(i=-33;i<34;i++)
        {a='';for(j=-69;j<34;j++){a+=M(c(
          j/33,i/33))?"□":"■"};console
                  .log(a)}

File: Kotlin/Mandelbrot.kt
                        import kotlin.math.sqrt
                fun main() { for (i in -33..34){for(j in -69..34){
            if(m(c(j.toDouble()/33,i.toDouble()/33)))print("□");else print("■")}
        println() } }; fun m(Z:c):Boolean{ val z:MutableList<c> =mutableListOf(c(0.0,0.0))
     for(i in 0..200) { val p:c=z.last()*z.last()+Z; if(p.s()>2)break;else z.add(p) };return z.size!=
        202 } class c{ var r=0.0;var i=0.0; constructor(re:Double, im:Double) {this.r=re;this.i=im}
           operator fun plus(z:c):c=c(this.r+z.r,this.i+z.i); operator fun times(z:c):c=
               c(this.r*z.r-this.i*z.i,this.r*z.i+this.i*z.r) fun s():Double=
                     sqrt(this.r*this.r+this.i*this.i) }
